spec/supa/commands/attribute_spec.rb

```diff
@@ -28,4 +28,29 @@
       it { expect(render_result).to eq('zaphod') }
     end
   end
+
+  context 'when :hide_when_nil option is set to true' do
```

Comment:
I think it needs its own spec (something like `attribute/options_spec.rb`), better move it out of the `attribute_spec`.